pkgs/tools/misc/journaldriver/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform, pkgconfig, openssl, systemd }:
+
+rustPlatform.buildRustPackage rec {
+  name        = "journaldriver";
```

Comment:
The version should be appended to the name, separated by a dash

Reply:
Fixed!